File: app/src/main/java/com/example/player/a1610aplayerdemo/mian/adapter/MainViewPagerAdapter.java
package com.example.player.a1610aplayerdemo.mian.adapter;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentStatePagerAdapter;
import com.example.player.a1610aplayerdemo.MyFragment;
import com.example.player.a1610aplayerdemo.SelectClassFragment;
import com.example.player.a1610aplayerdemo.StudyFragment;
import com.example.player.a1610aplayerdemo.VipFragment;

public class MainViewPagerAdapter extends FragmentStatePagerAdapter {
    private Fragment[] fragments = new Fragment[]{new SelectClassFragment(),new StudyFragment(),new VipFragment(),new MyFragment()};
    public MainViewPagerAdapter(FragmentManager fm) {
        super(fm);
    }

    @Override
    public Fragment getItem(int i) {
        return fragments[i];
    }

    @Override
    public int getCount() {
        return fragments.length;
    }
}

File: app/src/main/java/com/example/player/a1610aplayerdemo/ui/main/MainViewPagerAdapter.kt
package com.example.player.a1610aplayerdemo.ui.main

import android.support.v4.app.Fragment
import android.support.v4.app.FragmentManager
import android.support.v4.app.FragmentPagerAdapter
import com.example.player.a1610aplayerdemo.ui.main.choice.ChoiceFragment
import com.example.player.a1610aplayerdemo.ui.my.MyFragment
import com.example.player.a1610aplayerdemo.ui.study.StudyFragment
import com.example.player.a1610aplayerdemo.ui.vip.VIPFragment

class MainViewPagerAdapter : FragmentPagerAdapter {
    var fms:MutableList<Fragment> = ArrayList()
    constructor(fm:FragmentManager):super(fm){
        fms.add(ChoiceFragment())
        fms.add(VIPFragment())
        fms.add(StudyFragment())
        fms.add(MyFragment())
    }
    override fun getCount(): Int = fms.size

    override fun getItem(p0: Int): Fragment {
        return fms[p0]
    }
}